numba/core/event.py

```diff
@@ -423,3 +426,44 @@ def on_exit(*exc_details):
 
         start_event(kind, data=data)
         yield
+
+
+def _write_chrome_trace(rec):
+    """ If chrome tracing is enabled,
+        write event information (process/thread id, event name,
+        and duration time to json file.
+    """
+    import os
+    import threading
+
+    pid = os.getpid()
+    tid = threading.get_native_id()
```

Comment:
This is Python 3.8+ https://docs.python.org/3/library/threading.html#threading.get_native_id hence the code fails on Python 3.7. If you take a look at the output of `run_cmd` in https://github.com/numba/numba/pull/7537/files#diff-45abfee05930c5c2e330559e75b0b5ed4a7e1b56b1223f99674ad7a6f1b9a64fR20 you'll see:
```
 'line 440, in _write_chrome_trace\n'
 '    tid = threading.get_native_id()\n'
 "AttributeError: module 'threading' has no attribute 'get_native_id'\n"
```

Reply:
Thanks a lot @stuartarchibald. I fixed that.